src/algorithms/calorimetry/CalorimeterHitDigi.cc

```diff
@@ -59,8 +59,8 @@ void CalorimeterHitDigi::init(const dd4hep::Detector* detector, std::shared_ptr<
     tRes       = m_cfg.tRes / dd4hep::ns;
     stepTDC    = dd4hep::ns / m_cfg.resolutionTDC;
 
+    id_mask = 0;
```

Comment:
Can we at this point call this inverse mask with a docstring as to what the set bits are?

Reply:
Fixed